docker-compose.yml

```diff
@@ -90,12 +90,81 @@ services:
       OUTBOUND_EMAIL_ADAPTER_OPTS: "{\"api_key\":\"7da7d1cd-111c-44a7-b5ac-4027b9d230e5\"}"
       # Seeds
       STATIC_SEEDS: "true"
+      # Client info
+      USER_AGENT: ""
```

Comment:
This should be set to `iOS/12.5 (iPhone) connlib/0.7.412`.
See Device example in https://github.com/firezone/firezone/blob/cloud/elixir/README.md.